sdk/attestation/azure-security-attestation/CMakeLists.txt

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.13)
+project(azure-security-attestation LANGUAGES CXX)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../../cmake-modules")
+
+include(AzureVcpkg)
+include(AzureVersion)
+include(AzureCodeCoverage)
+include(AzureTransportAdapters)
+include(AzureDoxygen)
+include(AzureGlobalCompileOptions)
+include(AzureConfigRTTI)
+include(AzureBuildTargetForCI)
+
```

Comment:
What's a "map"? And how do I know if a component creates one?

Reply:
I have not added docs. Here's what to do: 

* Map files can only be produced when there's an executable unit of code (e.g. a test or sample program) not a static library.
* In relevant CMakeLists.txt files add the line (the `<target-name>` should be globally unique): 
```cmake
create_map_file(<target-name> <target-name>.map)
```

This will produce a map file with the name of your target. Example: https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/core/azure-core-test/CMakeLists.txt#L49

* The engineering system automatically scoops up all `**/*.map` files in the repo and uploads them as an artifact for the build. The folder structure is flattened so `.map` files end up in the same artifact folder. Make sure the target name is globally unique or you'll get undefined behavior.

Reply:
Oh - MAP files are the classic text dump of symbols? not a cmake construct? That makes so much more sense.